File: src/main/java/io/javalin/example/java/JavalinApp.java
package io.javalin.example.java;

import io.javalin.Javalin;
import static io.javalin.apibuilder.ApiBuilder.get;
import static io.javalin.apibuilder.ApiBuilder.post;

public class JavalinApp {

    private Javalin app = Javalin.create().routes(() -> {
        get("/users", UserController::getAll);
        post("/users", UserController::create);
        get("/ui", ctx -> ctx.html("<h1>User UI</h1>"));
    });

    public void start(int port) {
        this.app.start(port);
    }

    public void stop() {
        this.app.stop();
    }

}

File: src/main/kotlin/io/javalin/example/kotlin/JavalinApp.kt
package io.javalin.example.kotlin

import io.javalin.Javalin
import io.javalin.apibuilder.ApiBuilder.get
import io.javalin.apibuilder.ApiBuilder.post

class JavalinApp {

    val app = Javalin.create().routes {
        get("/users", UserController::getAll)
        post("/users", UserController::create)
        get("/ui") { it.html("<h1>User UI</h1>") }
    }

    fun start(port: Int) {
        this.app.start(port)
    }

    fun stop() {
        app.stop()
    }

}


